src/hotspot/share/opto/vectornode.hpp

```diff
@@ -1615,11 +1615,8 @@ class VectorRearrangeNode : public VectorNode {
 class VectorLoadShuffleNode : public VectorNode {
  public:
   VectorLoadShuffleNode(Node* in, const TypeVect* vt)
-    : VectorNode(in, vt) {
-    assert(in->bottom_type()->is_vect()->element_basic_type() == T_BYTE, "must be BYTE");
-  }
+    : VectorNode(in, vt) {}
```

Comment:
Can you add a comment above "class VectorLoadShuffleNode" to say what its semantics are?

Reply:
Done, I would refrain from renaming it right now.